peekingduck/pipeline/nodes/draw/image_processor.py

```diff
@@ -23,7 +23,25 @@
 
 
 class Node(AbstractNode):
-    """Node for changing image contrast and brightness"""
+    """Node for changing image contrast and brightness
+
+    The draw image processor node uses a mathematical algorithm to adjust the
+    contrast and brightness of the given image.
+
+    Inputs:
+
+        |img|
+
+    Outputs:
+        |img|
+
+    Configs:
+        brightness (:obj:`int`): **default = 0**
+            Adjusts the brightness of the image. Takes integer values -100 to 100.
+
+        contrast (:obj:`float`): **default = 1**
+            Adjusts the contrast of the image. Takes values 1 to 3.
```

Comment:
ranges could be defined in the defaults as well: https://github.com/aimakerspace/PeekingDuck/blob/313e530458ec6e3f1203518d0aab88d45b1277a0/peekingduck/pipeline/nodes/model/posenet.py#L72

Reply:
Wow nice. TIL.